src/Component/JsonSchemaGenerator/Tests/generated/open-banking-tracker/Model/OpenBankingTrackerApiAccessEnum.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file has been auto generated by Jane,
+ *
+ * Do not edit it directly.
+ */
+
+namespace Jane\Component\JsonSchemaGenerator\Tests\Generated\OpenBankingTracker\Model;
+
+enum OpenBankingTrackerApiAccessEnum: string
+{
+    case VERIFIEDTPP = 'verifiedTpp';
```

Comment:
I don't really like camelCase usage there, but IMO it's the responsibility of the JSON Schema writer and not ours. Its all fine :+1:

Reply:
done in the compiler